src/views/approvalsView.js

```diff
@@ -82,9 +103,53 @@ export default function ApprovalsTable({ history }) {
   const paginate = PageNumber => {
     setCurrentPage(PageNumber);
   };
+
+  const searchReducer = useSelector(state => state.searchReducer);
+  const allResults = [...searchReducer.data];
+
+	const [location, setLocation] = useState('');
+	const [name, setName] = useState('');
+	const [theStatus, setTheStatus] = useState('');
+  const [time, setTime] = useState('');
+
+  const [currentSearchPage, setCurrentSearchPage] = useState(1);
+  const [tripsSearchPerPage] = useState(8);
+
+  const indexOfLastSearchPage = currentSearchPage * tripsSearchPerPage;
+  const indexOfFirstSearchPage = indexOfLastSearchPage - tripsSearchPerPage;
+  const currentSearchTrips = allResults.slice(indexOfFirstSearchPage, indexOfLastSearchPage);
+
+  const paginateSearch = PageSearchNumber => {
+    setCurrentSearchPage(PageSearchNumber);
+  };
+
+	const handleChange = e => {
+		setLocation(e.target.value);
+		setName(e.target.value);
+		setTheStatus(e.target.value);
+		setTime(e.target.value);
+	};
+
+	const handleclick = () => {
```

Comment:
@erickyvand may you specify on which component does handleClick function handle? Applicable elsewhere